frontend/src/SignUp.tsx

```diff
@@ -81,19 +100,14 @@ export default function SignUp() {
                     />
                     <i className="eye" onClick={togglePasswordVisiblity}>
                     </i>
-                    <button type="submit" className = {"btn"}>Sign Up</button>
-                    {errorMessage.length > 0 ? (
-                        <div className="error-msg"> {errorMessage} </div>
-                    ) : null}
-                    {successMessage.length > 0 ? (
-                        <div className="success-msg">
-                            {successMessage}{' '}
-                            <button onClick={refreshPage} className="refresh">
-                                Refresh form
-                            </button>
-                        </div>
-                    ) : null}
-                </div>
+                    <Button type="submit" className = {"btn"}>Sign Up</Button>
+
+
+                    <div className="messages">
+                        {errorMessage()}
+                        {successMessage()}
```

Comment:
Correct me if im wrong but I think you use it without "()"

Reply:
Ich sehe die Meldungen auch nirgendswo